hs_core/models.py

```diff
@@ -4393,6 +4393,21 @@ def model_inputs(self):
                 self.create_element(element_model_name=element_name, **element[element_name])
 
 
+class TaskNotification(models.Model):
+    TASK_STATUS_CHOICES = (
+        ('progress', 'Progress'),
+        ('failed', 'Failed'),
+        ('aborted', 'Aborted'),
+        ('completed', 'Completed'),
+        ('delivered', 'Delivered'),
+    )
+    username = models.CharField(max_length=150, blank=True)
+    task_id = models.CharField(max_length=50, unique=True)
+    name = models.CharField(max_length=1000, blank=True)
+    payload = models.CharField(max_length=1000, blank=True)
+    status = models.CharField(max_length=20, choices=TASK_STATUS_CHOICES, default='pending')
```

Comment:
Have to change default to 'progress' now that pending status is removed.

Reply:
updated